packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/native/emptyTemplateJs/src/components/__tests__/HelloWorld.spec.jsx.ejs

```diff
@@ -0,0 +1,36 @@
+import { createElement } from "react";
+import { shallow } from "enzyme";
+import { Platform } from "react-native";
+
+import { HelloWorld } from "../HelloWorld";
+
+describe("HelloWorld", () => {
+    const render = (props) => shallow(<HelloWorld {...props} />);
+
+    it("it renders the structure correctly", () => {
+        const helloWorldProps = {
+            name: "Mendix",
+            style: [{ container: { borderColor: "white" }, label: { color: "black" } }]
+        };
+        const helloWorld = render(helloWorldProps);
+
+        expect(helloWorld).toMatchSnapshot();
+    });
+
+    describe("for iOS", () => {
```

Comment:
maybe do the same ios/android check?

Reply:
Added the same construction, although there is no actual difference in this case for iOS and Android.